operator/api/v1alpha1/operator_labels.go

```diff
@@ -0,0 +1,10 @@
+package v1alpha1
+
+const (
+	OperatorPrefix = "operator.kyma-project.io"
+	Separator      = "/"
+	OperatorName   = "module-manager"
+	OwnedByLabel   = OperatorPrefix + Separator + "owned-by"
+	OwnedByFormat  = "%s__%s"
+	WatchedByLabel = OperatorPrefix + Separator + "watched-by"
+)
```

Comment:
Labels can be revised / used from this file: https://github.com/kyma-project/module-manager/blob/main/operator/pkg/labels/component_labels.go

Reply:
I did not saw that such a pkg already existed, I will adapt